7-animation/2-css-animations/article.md

```diff
@@ -70,25 +70,25 @@ growing.onclick = function() {
 </script>
 ```
 
-Now let's cover animation properties one by one.
+Now, let's cover animation properties one by one.
 
 ## transition-property
 
-In `transition-property` we write a list of property to animate, for instance: `left`, `margin-left`, `height`, `color`.
+In `transition-property` we write a list of properties to animate, for instance: `left`, `margin-left`, `height`, `color`.
 
-Not all properties can be animated, but [many of them](http://www.w3.org/TR/css3-transitions/#animatable-properties-). The value `all` means "animate all properties".
+Not all properties can be animated, but [many of them can](http://www.w3.org/TR/css3-transitions/#animatable-properties-). The value `all` means "animate all properties".
 
 ## transition-duration
 
 In `transition-duration` we can specify how long the animation should take. The time should be in [CSS time format](http://www.w3.org/TR/css3-values/#time): in seconds `s` or milliseconds `ms`.
 
 ## transition-delay
 
-In `transition-delay` we can specify the delay *before* the animation. For instance, if  `transition-delay: 1s`, then animation starts after 1 second after the change.
+In `transition-delay` we can specify the delay *before* the animation. For instance, if  `transition-delay: 1s`, then the animation starts 1 second after the property change.
```

Comment:
```suggestion
In `transition-delay` we can specify the delay *before* the animation. For instance, `transition-delay: 1s`, starts the animation 1 second after the property change.
```

Reply:
Thanks. To be consistent maybe we should format the 2s as `2s` too.
...and `transition-duration` is `2s`, then...

Reply:
You're welcome! 👍 

> Thanks. To be consistent maybe we should format the 2s as `2s` too.
> ...and `transition-duration` is `2s`, then...

I missed that. I have added a new comment with this change.